index.js

```diff
@@ -90,9 +90,15 @@ module.exports.percyScreenshot = async function percyScreenshot(driver, name, op
       interceptor.restore();
     }
 
-    if (options && 'ignore_region_selenium_elements' in options) {
-      options.ignore_region_selenium_elements = await getElementIdFromElements(options.ignore_region_selenium_elements);
+    if (options) {
+      if ('ignore_region_selenium_elements' in options) {
```

Comment:
This is weird and looks inconsistent, ideally this should have been in camelCase considering this is js SDK, havent I can see that it was already not followed, but ideally our sdks should remain consistent with language specifics

Reply:
We support both camelCase and snakeCase. However for `ignore_region_selenium_elements` and `ignore_region_appium_elements` These were written in snake case. I did mention we should maintain these as well in camelCase for required languages.

Reply:
again we can explore supporting both `consider_region_selenium_elements` both in camelCase and snakeCase? CLI handles it that I am pretty sure.